### PR TITLE
Save and load steps,samples

### PR DESCRIPTION
**Pull request by Darkar25**
This made hypergan continue saving samples instead of overwriting them

### CODE REVIEW

File: hypergan/cli.py [hunk 1]
```diff
@@ -141,7 +144,11 @@ def train(self):
                 self.args.save_every > 0 and
                 i % self.args.save_every == 0):
                 print(" |= Saving network")
-                self.gan.save(self.save_file)
+                self.gan.save(self.save_file)   
+                self.create_path(self.advSavePath+'advSave.txt')
+                    with open(self.advSavePath+'advSave.txt', 'w') as the_file:
```
Comment (martyn) [inline hypergan/cli.py]: I'm getting a syntax error.  Indentation?

File: hypergan/cli.py [hunk 2]
```diff
@@ -43,6 +43,9 @@ def __init__(self, gan, args={}):
         self.sampler_name = args.sampler
         self.sampler = None
         self.validate()
+        
+        self.advSavePath = os.path.abspath("saves/"+self.config_name)+"/"
```
Comment (martyn) [inline hypergan/cli.py]: Can we rename this to self.save_path?

File: hypergan/cli.py (hunk 1 above)
Comment (Darkar25) [reply hypergan/cli.py]: Maybe...i wrote directly to github,not copied from my code...i have no syntax errors

File: hypergan/cli.py (hunk 2 above)
Comment (Darkar25) [reply hypergan/cli.py]: Yes

**Comment by mikkel** [comment]
I merged this into develop manually.

**Pull request closed without merging**